CancerStaging.fsh

```diff
@@ -14,13 +39,14 @@ Description:  "Abstract parent class for members of cancer staging panels. Cance
 * component 0..0
 * partOf only Reference(Procedure)
 * basedOn only Reference(ServiceRequest)
-* focus only Reference(PrimaryCancerCondition)
+// The following rule is causing a problem, possibly because of circular references (???). When left in, SUSHI dies silently and slowly.
+//* focus only Reference(PrimaryCancerCondition)
```

Comment:
Interestingly enough, this line does behave the way you state when uncommented, but it behaves differently on `master`. There, it reports errors as a result of circular references, and just finished with error output. For some reason, other changes on this branch also cause this new issue with a silent failure.

I'll make a note of this for while we try to fix the bug.